docs/app/pages/components/components-sections/tables/column-picker/column-picker.component.html

```diff
@@ -42,16 +46,22 @@
     <tr uxd-api-property name="selectedAriaLabel" type="(column: string, index: number) => string">
         Define a function to get the aria label of reorderable items in the selected column.
     </tr>
+    <tr uxd-api-property name="deselectedAriaLabel" type="(column: string, isExpanded: boolean) => string">
```

Comment:
I think you were going to rename this one?

Reply:
Sorry I didn't update it in the docs. Fixed now.